vendor/assets/javascripts/jquery_readyselector.js

```diff
@@ -0,0 +1,17 @@
+(function ($) {
```

Comment:
I gotcha. Could you put that GitHub URL at the head of this file for future reference?